screens/TabSettingsScreen.tsx

```diff
@@ -37,9 +45,14 @@ export default function TabSettingsScreen() {
         value={500}
         onSlidingComplete={setUserFilter}
       />
+      
+      <Pressable style={styles.button} accessibilityLabel="remove search radius limit" onPress={()=>setUserFilter(null}>
```

Comment:
I changed the button to a pressable so that the style is consistent across operating systems. Buttons create a weird styling difference for IOS and Android.